include/deal.II/lac/ginkgo_solver.h

```diff
@@ -99,8 +99,7 @@ namespace GinkgoWrappers
      * The @p solver_control object is the same as for other
      * deal.II iterative solvers.
      */
-    SolverBase(SolverControl &                solver_control,
-               std::shared_ptr<gko::Executor> executor);
+    SolverBase(SolverControl &solver_control, std::string exec_type);
```

Comment:
can you update what values this string argument can take? The documentation seems to still be written in the previous style.

Reply:
make the string a `const` reference. this is our usual style for non-scalar input arguments

same throughout below

Reply:
Thanks for the comments. I have made everything a `const &` now. The member variable does have the doumentation, but I have added it to the `SolverBase` class parameter doc here as well now.